v63004/inref/gtm8699.m

```diff
@@ -0,0 +1,26 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+gtm8699()
+
+	WRITE "    $VIEW(""STATSHARE"",""DEFAULT"")",!
+	WRITE "    "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+
+	WRITE "    VIEW ""STATSHARE"" ",!
```

Comment:
I would like to have a 2 region database file and do VIEW "STATSHARE":"DEFAULT" and verify that $VIEW now shows DEFAULT as sharing whereas it still shows the other region (AREG) as not sharing.